static/src/js/routes/SavedProjects/SavedProjects.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import { withRouter } from 'react-router-dom'
+import { Helmet } from 'react-helmet'
+
+import { getEnvironmentConfig } from '../../../../../sharedUtils/config'
+import SavedProjectsContainer from '../../containers/SavedProjectsContainer/SavedProjectsContainer'
+import AuthRequiredContainer from '../../containers/AuthRequiredContainer/AuthRequiredContainer'
+
+export class SavedProjects extends Component {
```

Comment:
We should make this a functional component like in `static/src/js/routes/Preferences/Preferences.js`